core/build.sbt

```diff
@@ -359,6 +359,10 @@ lazy val operator =
 
       publishArtifact in (Compile, packageDoc) := false,
       publishArtifact in (Compile, packageSrc) := false,
+      // skuber version 2.4.0 depends on akka-http 10.1.9 : hence overriding
+      // with akka-http 10.1.11 to use akka 2.6.3
+      // remove this override once skuber is updated
+      dependencyOverrides += "com.typesafe.akka" %% "akka-http" % Version.AkkaHttp,
```

Comment:
It uses `10.1.9` and we go straight to 10.1.11, I guess it is safe.

Reply:
FYI: we were already using 10.1.10 on other sub-projects.

@skonto Do you mean that Skuber also depends on those two artifacts?

Reply:
skuber does not depend on them. But skuber also depends on akka-stream .. not sure changing that to `2.6.3` will have any other impact ..

Reply:
@agemooij @debasishg we list the other too artifacts in Dependencies file for the core project. They both depend on the akka-http version are we sure that we get the right version of them?